src/app/(app)/(libarary)/library/page.tsx

```
@@ -0,0 +1,19 @@
+
+const page = () => {
+  const queryClient = getQueryClient();
+  void queryClient.prefetchInfiniteQuery(
+    trpc.library.getMany.infiniteQueryOptions({
+      limit: DEFAULT_LIMIT,
+    })
+  );
+  return;
+  <HydrationBoundary state={dehydrate(queryClient)}>
+    <LibraryView />
+  </HydrationBoundary>;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)